test/functional/loopTestDriver.py

```diff
@@ -42,27 +46,37 @@ def wait_for_element_enabled(self, element, timeout=10):
             .until(lambda e: element.is_enabled(),
                    message="Timed out waiting for element to be enabled")
 
-    def wait_for_element_attribute_to_be_false(self, element, attribute, timeout=10):
+    def wait_for_element_property_to_be_false(self, element, property, timeout=10):
+        # XXX We have to switch between get_attribute and get_property here as the
+        # content mode now required get_property for real properties of HTMLElements.
+        # However, in some places (e.g. switch_to_chatbox), we're still operating in
+        # a chrome mode. So we have to use get_attribute for these.
```

Comment:
Probably worth mentioning why we're operating in chrome mode, and point to a bug so that future readers can easily find out when we can get rid of that hack.